cmake/partition_manager.cmake

```diff
@@ -641,13 +641,17 @@ endif()
 # the primary slot is not known during the configure stage of the mcuboot
 # child image.
 if (CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY)
```

Comment:
Isn't this check also relevant if secondary slot is not located on external FLASH?

Reply:
I think that the check should be also done for non-external flash configurations,

Reply:
If the mcuboot secondary partition is stored in internal flash then its size is automatically set to the same value as mcuboot primary. Its only when you have the secondary slot in external flash that you don't get this set automatically, and hence we need this test. Note that in the next version 1.8 we will get the size of mcuboot secondary automatically also when external flash is used.